internal/apply/apply.go

```diff
@@ -127,3 +128,39 @@ func validateEnvironments(applyEnvironments []string) {
 		}
 	}
 }
+
+func summarizeAll(dir string, projectConfig projectconfig.ZeroProjectConfig) {
+	flog.Infof("Your projects and infrastructure have been successfully created.  Here are some useful links and commands to get you started:")
```

Comment:
I'm not sure about the `Infof` statements in this function.. just wanted to get placeholders in to show text at the beginning and end of the summary.